src/server/index.jsx

```diff
@@ -48,6 +48,7 @@ const server = express();
 server
   .disable('x-powered-by')
   .use('/data', express.static(dataFolderToRender))
+  .use('/public', express.static('public'))
```

Comment:
We serve public files in the line below using the razzle public directory, which moves `public` files to `build` for prod/serves from `public` for dev. This line will just serve everything from `public`, we should just use the line below.